node/cn/sc_backend.go

```diff
@@ -224,7 +229,7 @@ func (s *ServiceChain) APIs() []rpc.API {
 		}, {
 			Namespace: "klay",
 			Version:   "1.0",
-			Service:   downloader.NewPublicDownloaderAPI(s.protocolManager.downloader, s.eventMux),
+			Service:   downloader.NewPublicDownloaderAPI(dl, s.eventMux),
```

Comment:
How about adding `func (d *Downloader) Progress() klaytn.SyncProgress` to `protocolManagerDownloader`  interface? If possible, we can remove the line 212~216. :)

Reply:
Good idea! I will!